.travis.yml

```diff
@@ -22,7 +22,6 @@ env:
   matrix:
     # we recommend new addons test the current and previous LTS
     # as well as latest stable release (bonus points to beta/canary)
-    - EMBER_TRY_SCENARIO=ember-lts-2.12
```

Comment:
I've removed Ember 2.12 for now from our version support matrix for the time being because it was complaining about async side-effects of the components in the tests..but I didn't want to spend days on trying to refactor things. But we can always circle back around to debug 2.12 if theres a demand for it.